code/modules/mob/living/living.dm

```diff
@@ -1001,7 +799,7 @@
 				new path(src.loc)
 			butcher_results.Remove(path) //In case you want to have things like simple_animals drop their butcher results on gib, so it won't double up below.
 	visible_message("<span class='notice'>[user] butchers [src].</span>")
-	gib()
+	gib(0, 0, 1)
```

Comment:
we get organs but not bodyparts from harvest. Having to remove organs from limbs after harvesting would be too much work.